js/smart-side-nav.js

```diff
@@ -0,0 +1,56 @@
+namespacer('bcpl');
+
+bcpl.smartSideNav = (($, window) => {
+	const navLinksSelector = '.secondary-nav nav ul li a';
+	let activeWindow = window;
+
+	const getHrefWithoutQueryString = (href) => {
+		return href ? href.toLowerCase().split('?')[0] : undefined;
+	};
+
+	const compareNavLinks = (index, navLink) => {
+		const $navLink = $(navLink);
+
+		if (!$navLink.attr('href')) {
+			$navLink.removeClass('active');
+			return true;
+		}
+
+		const hrefWithoutQueryString = getHrefWithoutQueryString($navLink.attr('href'));
+		const locationUrlWithoutQueryString = getHrefWithoutQueryString(activeWindow.location.href);
+
+		if (locationUrlWithoutQueryString.endsWith(hrefWithoutQueryString)) {
+			$navLink.addClass('active');
+			return false;
+		}
+
+		$navLink.removeClass('active');
```

Comment:
lines 27 & 28 are repeated above, not sure if its worth a refactor or thought.

Reply:
Gave it a slight refactor; they're no longer repeated.